integration-tests/tests/utils/port_utils.py

```diff
@@ -0,0 +1,70 @@
+"""Functions for facilitating the port connections for the CLP package."""
+
+from clp_py_utils.clp_config import ClpConfig
+
+REQUIRED_PORTS = 10
```

Comment:
```suggestion
MAX_REQUIRED_PORTS = 10
```
since the actual number of required ports can change.